generators/client/templates/src/main/webapp/app/_app.config.js

```diff
@@ -1,49 +1,51 @@
-'use strict';
+(function() {
+  'use strict';
```

Comment:
I prefer setting `use strict` for the whole file rather than each function. Not sure what others think.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode